pkg/reconciler/revision/reconcile_resources.go

```diff
@@ -141,6 +142,9 @@ func (c *Reconciler) reconcilePA(ctx context.Context, rev *v1alpha1.Revision) er
 		if err != nil {
 			return perrors.Wrapf(err, "failed to create PA %q", paName)
 		}
+		if !pa.Status.IsReconciled() {
+			rev.Status.MarkResourcesUnavailable("PodAutoscalerHasNotReconciledYet", "PodAutoscaler with class "+pa.Annotations[autoscaling.ClassAnnotationKey]+" has not been reconciled yet")
```

Comment:
Why use `MarkResourcesUnavailable` here? That strikes me as wrong. We should use the `PodAutoscalerReconciled` condition instead.